### ISSUE TITLE
Snappymail irregularly resets users settings to server defaults

### ISSUE THREAD

**Issue by BeeFox-sys** [the-djmaze/snappymail#1842]
**Describe the bug**
Every so often the general settings for my user account reset to the server defaults, there is no information in logs that i can find, nor can i intentionally replicate it. however it happens about once a week.

**To Reproduce**
1. Change general settings from server defaults
2. Use snappymail as normal for a while
3. Settings have reset to server defaults

**Expected behavior**
Settings remain as user set options

**Please complete the following information:**
- Browser: Firefox 132.0
- IMAP daemon: unknown
- PHP version: 8.2.24
- SnappyMail Version: 2.38.2
- Mode: Stand Alone

**Debug/logging information**
[Read here how to log](https://github.com/the-djmaze/snappymail/wiki/FAQ#how-do-i-enable-logging)
No issues were seen in the logs at the warning/error level after following the guide on setting up the logs, i am continuing to record logs however due to the intermittent nature of the issue, it is hard to gather relevant logs as it can be days with the debug log going to gather the issue.

**Additional context**

**Comment by the-djmaze** [comment]
Check the date of files
- `data/_data_/_default_/storage/domain.name/user/settings`
- `data/_data_/_default_/storage/domain.name/user/settings_local`

You can also make them read-only and see what happens.

**Comment by BeeFox-sys** [comment]
I have managed to get the logs for the time that the settings file says it was changed last, there were no logs this whole day involving updating the settings of the user, until i was returning the settings to their intended configuration.

I have now removed write permissions on the `data/_data_/_default_/storage/domain.name/user/settings` file, and will report back either in a few weeks or if it still happens again

**Comment by BeeFox-sys** [comment]
This workaround works, issue persists however this is an ok work around for servers with only one user

**Comment by stef204** [comment]
I noticed this as well, where the font settings (per account) gets reset to default quite frequently.  I always have to go back to `settings|themes` and change the fonts again, back to what I want.  I am not inclined to use this workaround as read-only means every time you want to make a change, you´d have to log into docker and re-add the write permission, change the user setting inside the GUI, log back in again via console and remove the write perm, etc.  Not a great workaround.  Would be useful to get this fixed at some point.

**Comment by jas8522** [comment]
I can corroborate this. We have one user that requires spell check be enabled in the settings. They enable it and then at some seemingly random time in the future it becomes disabled. I'm betting that's this same issue here and the user hasn't changed anything other than this one config.

**Comment by BeeFox-sys** [comment]
I don't believe this to be a browser issue however it is possible, if for some reason the browser is asking to set everything to defaults.

**Comment by kernal64** [comment]
I have this issue for ages. I use to turn of the delay to mark messages as read by setting the value to "0" seconds. I thought the "0" is the problem, but now I see it's a general issue with user settings not persisting?!

**Comment by lfischer** [comment]
Same issue here.
This also affects the stored password, preventing automatic login using the mail credentials.

**Comment by umbertoB67** [comment]
Same issue here.

**Comment by umbertoB67** [comment]
While waiting for a fix that we hope our @the-djmaze can do as soon as possible, does anyone know if it is possible to modify some files to set different default values?
